src/domains/chart/utils/transformDataAttributes.ts

```diff
@@ -85,6 +85,7 @@ export interface StaticAttributes {
   units?: string
   unitsCommon?: string
   unitsDesired?: string
+  aggrMethod?: string
```

Comment:
can we use existing `method` attribute?

https://learn.netdata.cloud/docs/agent/web/gui/custom#data-grouping-method

Reply:
From the above documentation is seems that the grouping method (group param) is used when the Netdata server is requested to return fewer points for a time-frame.
aggrMethod is for aggregation across room multi nodes...

Reply:
@VLegakis please disregard, re: our discussion about aggregation across nodes